_posts/2016-03-07-2016-forman-survey-analysis.md

```diff
@@ -0,0 +1,342 @@
+
+Interestingly, larger numbers of users (>10) shows growth this year, up to
+nearly 25%. Hopefully this is due to continuing effort in our Roles,
+Authorization, and Locations/Organisations features. That said, a solid 50%
+remain in the 1-5 section, so keeping the option to disable or streamline these
+features is also important. This is relevant due to the discussion around
+Locations/Orgainisations at the moment.
+
+Finally for diversity, lets have a couple of bar charts:
+
+![Operatingsystems](/static/images/blog_images/2016-03-07-2016-forman-survey-analysis/operatingsystems.png)
+
+This year we allowed respondents to select more than one option, as "most" is
+subjective, especially when you put proxies and Foreman itself in the same
+question.
+
+Keeping this in mind, we see that Red Hat family distros remain constant at
+75%, but there's a marked increase in deb-based distros (up 5%) and in "Other"
+which is up 10%. It would be very interesting [to hear from those users][contact]
+about what OSs they are using!
+
+![Hammer vs API](/static/images/blog_images/2016-03-07-2016-forman-survey-analysis/hammer-vs-api.png)
+
+Hammer remains stable this year at just under 50%, and it's good to see APIv1
+declining. The real surprise here, however, is the "No" response jumping from
+just 6% to 32%. I have no data to suggest why this might be, but one conclusion
+is that the tooling we're providing to the newer users joining the community is
+now sufficient for their needs. If you've got a story about this,
+[let us know][contact]!
+
+Overall, the community seems diverse in many respects, with different sizes,
+ages, OSs and usecases. This is good news as we move forward, since it shows we
+have broad appeal to people just discovering Foreman today.
+
+## <a name="support"></a>Support and Docs
+
+Moving on then, we have a number of question related to how well we meet
+support needs of the community:
+
+![IRC and Lists](/static/images/blog_images/2016-03-07-2016-forman-survey-analysis/irc-and-lists.png)
+
+It's not very visible (thanks Google), but the vertical axis is, in order:
+
+* I know of the IRC channel, but don't use it
+* I use the IRC channel
+* I know of the mailing list, but don't use it
+* I use the mailing list
+* I wasn't aware of either
+
+The really important number for me here is the last one - 7% not aware of our
+primary support systems. That's both good (it's *only* 7%) and bad (how do we
+reach those people). For the others, so long as they know that the IRC channel
+and mailing lists exist, that's good - they can get help when they need it.
+
+To those who said they spend active time on the lists and on IRC - *thank you!*
+User-to-user support is one of the things that makes community leads go all
+warm and fuzzy and liable to buy rounds ;)
+
+![User Support](/static/images/blog_images/2016-03-07-2016-forman-survey-analysis/user-support.png)
+
+We have a great reputation for our IRC channel, and that's reflected here in
```

Comment:
Just a personal comment, I think it'd take a lot for people to see *any* mailing list or manual and say "5/5 awesome" :smile: